src/helpers/NavBarWrapper.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import NavBar from '../components/NavBar'
+
+function NavBarWrapper(Component) {
```

Comment:
@rcdjasmin Looking at the other contents now, I get a better idea now. A suggestion to make this better

```javascript
function NavBarWrapper(Component) {
    return function() {
        return (
            <React.Fragment>
                <NavBar/>
                <main>
                  <Component/>
                </main>
            </React.Fragment>
        )
    }
}
```

We can then remove `main` element repeated on the other page components